index.html

```diff
@@ -50,6 +50,231 @@ <h3 class="box3">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+        <p>
+             A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </p>   
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot1.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Tonic
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+          <p>
+              A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </p>  
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot3.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+           <p>
+                A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+           </p>
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
```

Comment:
As the previous reviewer said, we need to use only one `section` tag for the works section. Please follow this structure: 
```html
<section> // don't forget to add class for this section with 'display:grid'
  <div class="projects">...</div>
  <div class="projects">...</div>
  <div class="projects">...</div>
  <div class="projects">...</div>
<section>
```
(Notice that this section is for works section. We will implement one section and we will add all our projects inside it as `div`s)

Reply:
Addressed 🙌